sla_per_use_system_test.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright (c) 2016 ScyllaDB
+
+from avocado import main
+from performance_regression_test import PerformanceRegressionTest
+from sdcm.db_stats import PrometheusDBStats
+import time
+
+
+class SlaPerUserTest(PerformanceRegressionTest):
+    """
+    Test SLA per user feature using cassandra-stress.
+    """
+
+    STRESS_WRITE = 'cassandra-stress write cl=ALL n={n} -port jmx=6868 -mode cql3 native user={user} ' \
+                   'password={password} -rate threads={threads}'
+    STRESS_READ = 'cassandra-stress read cl=ALL duration={duration} -port jmx=6868 -mode cql3 native user={user} ' \
+                  'password={password} -rate threads={threads} -pop seq={seq}'
+    USER = 'cassandra'
+    PASSWORD = 'cassandra'
+    DEFAULT_SHARES = 1000
+    VALID_DEVIATION_PRC = 10
+    WAIT_FOR_LOAD = 300 # seconds
+
+    def __init__(self, *args, **kwargs):
```

Comment:
why do we need `*args, **kwargs` here?

Reply:
We need because of avocado transfer different parameters to the test class. 
Without it the test can't be run